packages/block-library/src/post-terms/edit.js

```diff
@@ -89,14 +90,25 @@ export default function PostTermsEdit( {
 					} }
 				/>
 			</BlockControls>
+			<InspectorAdvancedControls>
+				<TextControl
+					autoComplete="off"
+					label={ __( 'Separator' ) }
+					value={ separator || '' }
+					onChange={ ( nextValue ) => {
+						setAttributes( { separator: nextValue } );
+					} }
+					help={ __( 'Enter character(s) used to separate terms.' ) }
+				/>
+			</InspectorAdvancedControls>
 			<div { ...blockProps }>
 				{ isLoadingTermLinks && <Spinner /> }
 
 				{ hasTermLinks &&
 					! isLoadingTermLinks &&
 					termLinks.reduce( ( prev, curr ) => [
 						prev,
-						' | ',
+						separator || ', ',
```

Comment:
I feel like this is a bit inconsistent because a user could not have filled the attribute but a `,` is showing. This is more obvious when we insert the block and we add `,` as a separator which removes the extra space.

I think it might be better to have the `,` as a default value in `block.json` and here just add an `empty space`, if separator has no value.